data/lang/module-cargorun/en.json

```diff
@@ -213,35 +217,35 @@
   },
   "HOWMUCH_1": {
     "description": "regarding mass, always more than 1t",
-    "message": "{amount} metric tons."
+    "message": "Yes it is. The total amount is {amount} metric tons."
   },
   "HOWMUCH_2": {
     "description": "regarding mass, always more than 1t",
-    "message": "{amount}t"
+    "message": "Yes, we can split the cargo. I have {amount}t in total."
   },
   "HOWMUCH_3": {
     "description": "regarding mass, always more than 1t",
-    "message": "The mass is {amount}t."
+    "message": "Yes. The entire mass is {amount}t."
   },
   "HOWMUCH_4": {
     "description": "regarding mass, always more than 1t",
-    "message": "There are {amount}t."
+    "message": "There are {amount}t. We can split the cargo run if you want."
```

Comment:
Just a personal opinion, I think HOWMUCH_4 and HOWMUCH_2 are basically the same, i.e. a player seeing two adverts, with each flavor will probably not notice they're technically different. (Generally, I'd opt for having fewer flavours if they're this similar)
Action: none required, I just felt like pointing out my view (which could be wrong) on this subject.

Reply:
Yes, sounds very similar. I will remove one.